packages/ui-components/src/assets/styles/kv-typography.scss

```diff
@@ -238,4 +270,5 @@
 	word-break: normal;
 	word-wrap: normal;
 	line-height: 1.125;
```

Comment:
can we change this to `line-height: 1.3125rem;` to match our DS?

Reply:
done